app/policies/article_policy.rb

```diff
@@ -1,10 +1,10 @@
-class ArticlePolicy
+class ArticlePolicy < BasePolicy
   def initialize(user, article)
-    @user = user
+    self.user = user
```

Comment:
Here i try to 'hide' this attribute from 'outer world', making his get'ter and set'ter protected. You asking me 'why here \@user? This attribute can be internal'. So, i try make this attribute internal with protected attributes.